src/sync/transforms/pull-request.ts

```diff
@@ -27,8 +35,7 @@ export const transformPullRequest =  async (payload: Payload, prDetails: Octokit
 	}
 
 	return {
-		id: repository.id.toString(),
-		name: repository.full_name,
+		... transformRepositoryDevInfoBulk(repository, gitHubBaseUrl),
```

Comment:
Drag-n-drop replacement (with exception of changing repository.name to repository.full_name - this is what we have in other places), not feature-flagging
<img width="1423" alt="image" src="https://user-images.githubusercontent.com/20631664/194985633-8e194799-f949-4dfb-bc0d-4683361664fe.png">
(on the left - removed lines, on the right - the mapping inside the function)